qiskit/providers/ibmq/ibmqsingleprovider.py

```diff
@@ -66,11 +66,12 @@ def backends(self, name=None, filters=None, **kwargs):
 
         return filter_backends(backends, filters=filters, **kwargs)
 
-    def _authenticate(self, credentials):
+    def _authenticate(self, credentials, verify=True):
```

Comment:
Instead of receiving `verify` as a parameter, can you use `credentials.verify`? The overall idea is that the `credentials` objects act as "containers" that hold all the information relevant for connecting to the API, being passed around in the upper layers, and being inspected and used precisely by this function (so the lower layers deal with the individual elements).